test/python/expected/upgrade_validation/expected_dependency.out

```diff
@@ -218,8 +218,10 @@ Function sys.babelfish_update_job(integer,character varying)
 Function sys.babelfish_waitfor_delay(text)
 Function sys.babelfish_waitfor_delay(timestamp without time zone)
 Function sys.bbf_binary_cmp(sys.bbf_binary,sys.bbf_binary)
+Function sys.bbf_binary_varbinary_cmp(sys.bbf_binary,sys.bbf_varbinary)
```

Comment:
This means that we don't have dependent objects for this function in the prepare script. In other words, we should not add more entries into this file. Please add views using those functions in the prepare script.

Reply:
bbf_binary_varbinary_cmp and bbf_varbinary_binary_cmp can't be get rid of using create view